.github/workflows/docker.yml

```
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        php-versions: ['7.1', '7.2', '7.3', '7.4']
+        php-versions: ['7.2', '7.3', '7.4', '8.0']
```

```suggestion
        php-versions: ['7.2', '7.3', '7.4', '8.0', '8.1']
```
ping @boubkerbribri 😉